packages/backend-storage/src/storage_container_entry_generator.test.ts

```diff
@@ -150,7 +150,8 @@ void describe('StorageGenerator', () => {
       const template = Template.fromStack(stack);
       template.hasResourceProperties('Custom::S3BucketNotifications', {
         BucketName: {
-          Ref: 'testNameBucketB4152AD5',
+          // eslint-disable-next-line spellcheck/spell-checker
+          Ref: 'testnameBucketEF49125A',
```

Comment:
If this will be common enough (and I guess it will if this is naming convention we have) I'd just add to dictionary.

Reply:
updated.